src/braket/ahs/atom_arrangement.py

```diff
@@ -52,7 +52,7 @@ def add(
         Returns:
             AtomArrangement: returns self (to allow for chaining).
         """
-        self._sites.append(AtomArrangementItem(coord, site_type))
+        self._sites.append(AtomArrangementItem(tuple(coord), site_type))
```

Comment:
I'm not sure about this. The type hint already says tuple. It should either get converted to tuple before it's passed in, or maybe we should just support add(np.ndarray) ?

Reply:
What do you mean by this? "It should either get converted to tuple before it's passed"

1. If you believe this conversion is already happening automatically, I have to disagree. Without this change in line 55, the new test (test_converting_numpy_array_sites_to_ir) in this PR does not pass.

2. If you believe we must implement `add` in such a way that this conversion happens, then I believe I have done exactly that.

Reply:
I mean that the caller could call:
```
    for site in sites:
        register.add(tuple(site))
```

Reply:
However, I'm not opposed to the idea of adding a convenience method for adding an np.ndarray.